lib/fruitbot/streampusher_api.ex

```diff
@@ -1,35 +1,68 @@
 defmodule Fruitbot.StreampusherApi do
   def user_search(query) do
-  end
+    case HTTPoison.get!("https://datafruits.streampusher.com/api/djs/#{query}.json?name=#{query}") do
+      %HTTPoison.Response{status_code: 200, body: body} ->
+        response = Jason.decode!(body)
+        data = response["data"]
+        username = data["attributes"]["username"]
+        url = "https://datafruits.fm/djs/#{username}"
+        "Datafruit found: #{url}"
+      %HTTPoison.Response{status_code: 404} ->
```

Comment:
the API is still not responding with 404.  i vote to remove this and just have the catch all below `_ -> `

Reply:
Hmmm seems to be working for for the /djs endpoint
![line_20230406_002452](https://user-images.githubusercontent.com/66243/230304822-91348966-ecdd-44a4-af8d-61222c76da33.png)